composer.json

```diff
@@ -65,7 +65,7 @@
         "symfony/filesystem": "^5.4 || ^6.0 || ^7.0",
         "symfony/finder": "^5.4 || ^6.0 || ^7.0",
         "symfony/process": "^5.4 || ^6.0 || ^7.0",
-        "thecodingmachine/safe": "^2.1.2",
```

Comment:
`thecodingmachine/safe` has fixes in master, but didn't create a new tag since 2023